CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 3.1.9
+  - Change default sincedb path to live in `{path.data}/plugins/inputs/s3` instead of $HOME.
+    Upgrades should not be negatively impacted because the old file in $HOME should be detected
```

Comment:
@jordansissel do we actually want a sentence with two "should" in the changelog? Just a suggestion, but I'd simply leave out the second sentence here.

Reply:
I want to put the automatic-upgrade detail in here in case users ask we have somewhere to point them.

Any thoughts on rewording this? "Older installs will be automatically migraded from the old $HOME to new `{path.data}/...` default" ?

Reply:
@jordansissel yea that sounds better (assuming "migraded" => "migrated" :))

Reply:
I rewrote this line now.